src/main/webapp/app/shared/util/tab.constants.ts

```diff
@@ -11,6 +11,11 @@ export const eventTabList: ITabInfo[] = [
     tabTranslateKey: 'clubmanagementApp.eventChecklist.tab',
     tabRoute: '/event/checklist'
   },
+  {
+    tabName: 'Activities',
+    tabTranslateKey: 'clubmanagementApp.eventActivity.tab',
+    tabRoute: '/entity/event-activity/event/1'
```

Comment:
`/entity/event-activity/event/1`
are you sure this URL should be hardcoded?

Reply:
yes you are right this cannot be hardcoded
@kachun333 can I create a new issue and resolve this in another branch?
with all event module tab update

Reply:
@HowardYaw okay, no problem